packages/store-ui/src/utils/throttle.ts

```diff
@@ -0,0 +1,23 @@
+const throttle = <T extends any>(cb: (...args: any) => T, wait = 166) => {
```

Comment:
Is the typing working here? I mean, the returned function will have the same typings as `cb`?

Reply:
Nice catch ! I've just fixed it